frontend/packages/console-plugin-sdk/src/codegen/active-plugins.ts

```diff
@@ -40,5 +43,117 @@ export const getActivePluginsModule = (pluginPackages: PluginPackage[]) => {
     export default activePlugins;
   `;
 
-  return output.replace(/^\s+/gm, '');
+  return trimStartMultiLine(output);
+};
+
+/**
+ * Important: keep this in sync with `getActivePluginsModule` above.
+ */
+export const loadActivePluginsForTestPurposes = (
+  pluginPackages: PluginPackage[],
+  dynamicExtensionHook: (pkg: PluginPackage) => Extension[] = _.constant([]),
+) => {
+  const activePlugins: ActivePlugin[] = [];
+
+  for (const pkg of pluginPackages) {
+    activePlugins.push({
+      name: pkg.name,
+      extensions: [
+        // eslint-disable-next-line
+        ...require(`${pkg.name}/${pkg.consolePlugin.entry}`).default,
```

Comment:
This `require` call triggers multiple ESLint rule violations:

```
/home/vszocs/go/src/github.com/openshift/console/frontend/packages/console-plugin-sdk/src/codegen/active-plugins.ts
  62:12  error  Calls to require() should use string literals  import/no-dynamic-require
  62:12  error  A `require()` style import is forbidden        @typescript-eslint/no-require-imports
```

Neither of them should apply, since this code is executed in Node.js env. at webpack build time.

I'll update our `console/node-typescript-prettier` ESLint plugin and remove all relevant `// eslint-disable-next-line` comments on `require` calls.